ogm/index.html.haml

```diff
@@ -17,20 +17,27 @@ project_hero_partial: project/ogm/hero-text.html.haml
 
     == Wide range of backends
 
-    OGM talks to NoSQL backends via store-specific dialects. Currently there is support for
+    OGM talks to NoSQL backends via store-specific dialects. In OGM proper, there is support for
```

Comment:
I think I never heard the expression:`In OGM proper`. what do you mean by it?

Maybe we can just remove the `Currently,` and use `It supports ...`